packages/web/hooks/ui-config/use-trade-token-in-config.ts

```diff
@@ -20,7 +21,9 @@ export function useTradeTokenInConfig(
   chainGetter: ChainGetter,
   osmosisChainId: string,
   bech32Address: string,
-  queriesStore: QueriesStore<[CosmosQueries, CosmwasmQueries, OsmosisQueries]>,
+  queriesStore: QueriesStore<
```

Comment:
I actually found out this can be simplified if we use the IQueriesStore interface. Then, only add the queries in that list that are actually needed. (It's more generic).

https://github.com/osmosis-labs/osmosis-frontend/blob/a3b18b99dd7026ed7acd65ec3553abf0d4486cfe/packages/stores/src/account/index.ts#L31-L32

Should fix the build errors elsewhere as well.

Reply:
Nice catch 🙌  

Updated